conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationMetadataEntitySetsService.kt

```diff
@@ -108,10 +98,33 @@ class OrganizationMetadataEntitySetsService(
         }
     }
 
-    fun isFullyInitialized(): Boolean = this::organizationMetadataEntityTypeId.isInitialized &&
-            this::datasetEntityTypeId.isInitialized && this::columnsEntityTypeId.isInitialized &&
-            this::omAuthorizedPropertyTypes.isInitialized && this::datasetsAuthorizedPropertyTypes.isInitialized &&
-            this::columnAuthorizedPropertyTypes.isInitialized && this::propertyTypes.isInitialized
+    fun isFullyInitialized(): Boolean = this::organizationMetadataEntityTypeId.isInitialized
+            && this::datasetEntityTypeId.isInitialized
+            && this::columnsEntityTypeId.isInitialized
+            && this::omAuthorizedPropertyTypes.isInitialized
+            && this::datasetsAuthorizedPropertyTypes.isInitialized
+            && this::columnAuthorizedPropertyTypes.isInitialized
+            && this::propertyTypes.isInitialized
+
+    fun initializeOrganizationMetadataEntitySets(organizationId: UUID) {
+        val adminAclKey = organizations.executeOnKey(
+            organizationId,
+            OrganizationReadEntryProcessor { it.adminRoleAclKey }
+        ) as AclKey
+
+
+        initializeOrganizationMetadataEntitySets(securePrincipalsManager.getRole(organizationId, adminAclKey[1]))
```

Comment:
This should probably be using `PrincipalsMapManager.lookupRole` (as PrincipalsMapManager is intended to be a lightweight read layer over the mapstore and `SecurePrincipalsManager` is intended for more heavy-duty/workflow-specific computation on top of that) -- and that would also let you just pass `adminAclKey` directly instead of having to do hacky deconstruction of the AclKey